src/main/steps/common/components/nationality.ts

```diff
@@ -91,7 +100,7 @@ export const nationalityFields = (userCase: Partial<Case>, fieldPrefix: FieldPre
                   type: 'summarylist',
                   values: [],
                   rows: mapSummaryListContent(
-                    userCase[`${fieldPrefix}AdditionalNationalities`]!,
+                    userCase[`${fieldPrefix}AdditionalNationalities`] || [],
```

Comment:
Or condition is not required since this component will only be rendered when `userCase[`${fieldPrefix}AdditionalNationalities`]?.length` is true

Reply:
Better practice to use this rather than null assertion, either eslint or prettier was complaining. Please let me know if you would still like me to change